modules/nw-proxy-configure-object.adoc

```diff
@@ -109,7 +109,7 @@ spec:
 Port numbers are only supported when configuring IPv6 addresses. Port numbers are not supported when configuring IPv4 addresses.
 ====
 +
-Preface a domain with `.` to match subdomains only. For example, `.y.com` matches `x.y.com`, but not `y.com`. Use `*` to bypass proxy for all destinations.
+If your `noproxy` field includes subdomains or CIDR blocks, you must also explicitly specify the IP addresses you want to exclude in order to correctly route the request. This is because the cluster will not wait for DNS to return the IP address before assigning the route connection.
```

Comment:
```suggestion
If your `noproxy` field includes subdomains or CIDR blocks, you must also explicitly specify the IP addresses you want to exclude in order to correctly route the request. This is because the cluster does not wait for DNS to return the IP address before assigning the route connection.
```

IMO this may use "does"/present instead of "will"/[future tense](https://redhat-documentation.github.io/supplementary-style-guide/#will)